src/components/MainLayout/MainPage/CommonGameArea/GameBoard/GameBoardPart/PanelRow/WeatherPart.tsx

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components';
+import { colors } from '../../../../../../../theme/variables';
+import gameAreaBlock from '../../../../../../../assets/battlefield/field-item.png';
+
+export const WeatherPart = styled.div`
```

Comment:
Can you please rename weatherPart to SpecialCardsPart? I checked and this area is used just for special cards, 
![image](https://github.com/solovevayaroslavna/simbirgwent/assets/90199600/198ff5a9-12d1-4764-a1d0-35db064c9313)
The place for weather - left area between players
![image](https://github.com/solovevayaroslavna/simbirgwent/assets/90199600/3af96d7b-f482-471f-b59f-dd76c0aba55b)

Reply:
Ok, no problem